store/sdk/src/test/java/org/apache/carbondata/sdk/file/AvroCarbonWriterTest.java

```diff
@@ -29,16 +29,27 @@
 import org.apache.avro.generic.GenericData;
 import org.apache.commons.io.FileUtils;
 import org.apache.commons.lang.CharEncoding;
+import org.junit.After;
 import org.junit.Assert;
+import org.junit.Before;
 import org.junit.Test;
 
 import tech.allegro.schema.json2avro.converter.JsonAvroConverter;
 import org.apache.avro.Schema;
 
-
 public class AvroCarbonWriterTest {
   private String path = "./AvroCarbonWriterSuiteWriteFiles";
 
+  @Before
+  public void cleanFile() {
+    assert (TestUtil.cleanMdtFile());
```

Comment:
Is there another PR to remove the creation of system folder when user uses SDK to write data?

Reply:
I think pr 2246 can solve it

Reply:
PR 2246 fix another problem, can't solve this one. @ravipesala

Reply:
@jackylk This PR dependency on PR2246, the first commit it cherry-pick from PR2246.
After pr2246 merged, this pr need rebase.